README.md

```diff
@@ -0,0 +1,88 @@
+# Experian Data Quality - Postman Collections
+
+A series of Postman collections for Experian Data Quality's Contact Data Management APIs.
+
+Full API documentation can be found at [edq.com/documentation/apis](http://edq.com/documentation/apis).
+
+Postman is a GUI REST client. By importing the collections you can explore our APIs and get a better understanding of the requests and responses.
+
+## Getting Started
+
+You will need an authentication token to make any requests. For existing customers, you can get your token from our [Self Service Portal](https://portal.experianmarketingservices.com/). If you're not an existing customer, visit [edq.com](http://edq.com) for more information about our products and how to contact us.
+
+To get started with the Postman collections you will need to download the Postman tool from [getpostman.com/postman](https://www.getpostman.com/postman).
+
+With Postman installed, you can clone this repository or download the individual files. You can then import the files into Postman and start making your requests.
```

Comment:
Check [this](https://github.com/postmanlabs/postman-app-support/issues/751) out. The implication is that these days it's possible to provide a link that'll open up Postman with particular settings. In particular, it *might* be possible to provide a URL that'll open up postman (assuming it's installed) and immediately import the collection from its raw github URL. Definitely beyond the scope of this PR, but worth investigating.